.github/workflows/ci-actions.yml

```diff
@@ -11,13 +12,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: [11, 16]
+        java: [11, 17]
```

Comment:
Just for the record, `[11, 17]` is not an interval, it's an alternative syntax for lists. It is equivalent to this:

```yaml
jobs:
  build:
    name: "..."
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix:
        java:
        - 11
        - 17
...
```